### ISSUE TITLE
Brave search mcp server error: fetch is not defined

### ISSUE THREAD

**Issue by juanmacedan1co** [modelcontextprotocol/servers#300]
**Describe the bug**
I have configured the brave search mcp server on my mac, but when Claude tried to used got **Error: fetch is not defined**

**To Reproduce**
1. Configure the brave search mcp server like this
"brave-search": {
        "command": "npx",
        "args": [
          "-y",
          "@modelcontextprotocol/server-brave-search"
        ],
        "env": {
          "BRAVE_API_KEY": "BSASaoHXXXXXXXXXXXX"
        }
      }
2. Restart Claude Desktop app
3. look for the MCP available tools
4. got the brave_web_search tools listed
5. ask about something like: look something about steve jobs 
6. Claude tried to do
{
  `count`: 5,
  `query`: `Steve Jobs biography achievements Apple history`
}

**Expected behavior**
To use the results from the brave search API

**Logs**
If applicable, add logs to help explain your problem.

**Additional context**
My current Claude Desktop version is Version 0.7.5 (0.7.5)

**Comment by juanmacedan1co** [comment]
the fix was to include the correct node version in the ENV 
PATH=/Users/username/.nvm/versions/node/v20.18.0/bin:/usr/local/bin:/usr/bin:/bin

**Comment by wolf019** [comment]
Thanks for reporting this issue! I encountered the same "Error: fetch is not defined" problem.

The solution that worked for me was updating the Node version in the PATH environment variable in the claude_desktop_config.json:

{
  "mcpServers": {
    "brave-search": {
      "command": "npx",
      "args": [
        "-y",
        "@modelcontextprotocol/server-brave-search"
      ],
      "env": {
        "BRAVE_API_KEY": "your-api-key",
        "PATH": "/Users/username/.nvm/versions/node/v20.18.0/bin:/usr/local/bin:/usr/bin:/bin"
      }
    }
  }
}

Key points:
1. Make sure to use a recent Node version (I updated from v16 to v20)
2. Include the full PATH with all system directories
3. Restart Claude Desktop after making these changes

This resolved the fetch not defined error and now the Brave search functionality works perfectly!
